common/rand.s

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Filename casing is important on Linux, please rename to `rand.S` (refer to our other Linux assembly files)

Reply:
Changed